utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -93,6 +93,12 @@ class HtmlParser private constructor(
       htmlContent = modifiedHtmlContent
     } else {
       htmlContentTextView.textDirection = View.TEXT_DIRECTION_LTR
+      val regex = Regex("""<oppia-noninteractive-image [^>]*>.*?</oppia-noninteractive-image>""")
+      val modifiedHtmlContent = rawString.replace(regex) {
+        val oppiaImageTag = it.value
```

Comment:
DRY: This code is a duplicate of the code on line 88.

Reply:
Its not exactly duplicate. Regex in line no 88 makes image align explicitly using in-line CSS (Refer #5212 PR) but this doesn't.

Reply:
Even so, the regex variable is a duplicate and can be extracted outside the block to be used in both places.

Reply:
okay.